src/caret_analyze/infra/lttng/ros2_tracing/processor.py

```diff
@@ -987,9 +999,9 @@ def _handle_callback_group_add_subscription(
         callback_group_addr = get_field(event, 'callback_group_addr')
         subscription_handle = get_field(event, 'subscription_handle')
 
-        callback_group_addr = self._callback_group_addr_remapper.get_nearest_object_id(
+        callback_group_addr = self._remapper.callback_group_addr_remapper.get_nearest_object_id(
             callback_group_addr, event)
-        subscription_handle = self._subscription_handle_remapper.get_nearest_object_id(
+        subscription_handle = self._remapper._subscription_handle_remapper.get_nearest_object_id(
```

Comment:
Fixing direct access to private member.
```suggestion
        subscription_handle = self._remapper.subscription_handle_remapper.get_nearest_object_id(
```